src/interface/app.yaml

```diff
@@ -0,0 +1,302 @@
+openapi: 3.0.1
+info:
+  title: PISP Demo Server Open API
+  description: Based on API Definition.docx Version 1.1 that was confirmed on 2020-05-19.
```

Comment:
where is this API Definition.docx file?

Reply:
Oh, I actually just copied the description from the Mojaloop's Open API template, so I believe it refers to Mojaloop API Definition. Shall I just change this into a different description?

Reply:
I'd imagine that this is very confusing for someone without context of the template. In fact I couldn't even find that file. Maybe you can remove this.

Reply:
okay, updated